plugins/UM3NetworkPrinting/resources/qml/PrintJobInfoBlock.qml

```diff
@@ -27,6 +133,9 @@ Item
     Rectangle
     {
         id: background
+
+        height: (cardHeight + alertHeight()) * screenScaleFactor
```

Comment:
Using dynamic heights is strongly discouraged by Qt, since it's much slower. Anchors are the prefered solution.

Reply:
This is already being fixed in my branch from this branch where I'm rewriting the Print Info Block from scratch to be much cleaner. Don't spend more time reviewing this for now.